README.md

```diff
@@ -21,8 +20,14 @@ var results = AuthorModel.find({name: "Bob"});
 console.log(results);
 ```
 
+```text
+The example provided needs to have a callback function so that the results of the query can be passed into the response.
+```
 ```js
-// Your answer...
+var results = AuthorModel.find({name: "Bob"}, function(err,data){
+  console.log(data);
+});
```

Comment:
👍 I'll give it to you, but it doesn't explain why `data` isn't an instance of the Author model.